opencga-app/app/scripts/azure/arm/daemonvm/azuredeploy.json

```diff
@@ -0,0 +1,185 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": ""
+            }
+        },
+        "_artifactsLocationSasToken": {
+            "type": "string",
+            "metadata": {
+                "description": ""
+            }
+        },
+        "subnetName": {
+            "type": "string",
+            "defaultvalue": "default"
+        },
+        "virtualNetworkId": {
+            "type": "string"
+        },
+        "publicIpAddressType": {
+            "type": "string",
+            "defaultvalue": "Dynamic"
+        },
+        "publicIpAddressSku": {
+            "type": "string",
+            "defaultvalue": "Basic"
+        },
+        "osDiskType": {
+            "type": "string",
+            "defaultvalue": "Premium_LRS"
+        },
+        "virtualMachineSize": {
+            "type": "string",
+            "defaultvalue": "Standard_D2s_v3"
+        },
+        "adminUsername": {
+            "type": "string",
+            "defaultValue": "opencgaadmin"
+        },
+        "adminPassword": {
+            "type": "securestring"
+        },
+        "hdInsightSshDns": {
+            "type": "string"
+        },
+        "hdInsightSshUsername": {
+            "type": "string"
+        },
+        "hdInsightSshPassword": {
+            "type": "securestring"
+        }
+
+    },
+    "variables": {
+        "vnetId": "[parameters('virtualNetworkId')]",
+        "subnetRef": "[concat(variables('vnetId'), '/subnets/', parameters('subnetName'))]",
+        "cloud-init": [
+
+            "#cloud-config",
+            "package_upgrade: true",
+            "package_reboot_if_required: true",
+            "",
+            "packages:",
+            "- docker.io",

```

Comment:
Nit: 'docker.io' is the Ubuntu repository version of Docker so will lag behind the official repo from docker.

Reply:
Believe this will give 17.x while official docker repo is now 18.x

Reply:
It's changed recently. Good explanation... https://www.collabora.com/news-and-blog/blog/2018/07/04/docker-io-debian-package-back-to-life/. It was just easier, happy to change.

Reply:
I'd use something like 
```
runcmd:
- apt-get update
- curl -fsSL https://get.docker.com/ | sh
- curl -fsSL https://get.docker.com/gpg | sudo apt-key add -
```

cc @jjcollinge as he's picking this up

Reply:
as we don't know how up-to-date the deb package will be kept